src/Snippets/Docs/Migration.Wrap.cs

```diff
@@ -11,9 +11,9 @@ public static void PolicyWrap_V7()
         IAsyncPolicy timeoutPolicy = Policy.TimeoutAsync(TimeSpan.FromSeconds(3));
 
         // Wrap the policies. The policies are executed in the following order (i.e. Last-In-First-Out):
-        // 1. Retry
-        // 2. Timeout
-        IAsyncPolicy wrappedPolicy = Policy.WrapAsync(timeoutPolicy, retryPolicy);
+        // 1. Retry <== outer
+        // 2. Timeout <== inner
+        IAsyncPolicy wrappedPolicy = Policy.WrapAsync(retryPolicy, timeoutPolicy);
```

Comment:
Is this true though? Isn't timeout policy executed first here?

Reply:
https://dotnetfiddle.net/znmcPA

Reply:
You can see the order in case of V7 and V8.

Reply:
true, i got it wrong it my head then. I think we should also update/drop this note then:

Reply:
I will update the PR later today or on Monday